src/loops/programs_with_repetition/easy/q5/MirrorSeven.java

```diff
@@ -0,0 +1,36 @@
+package loops.programs_with_repetition.easy.q5;
+
+/*
+ * Write a program to keep asking the user to enter a number until they enter a palindrome number that is also
+ * divisible by 7 (i.e. it can be read the same forwards and backwards)
+ * 
+ * Sample output:
+ *    Enter an integer: 
+ *    777
```

Comment:
The sample output should be changed to show that "Enter an integer:" should be output each time the enter a number until the program encounters a valid number.